docs/kms_secretlock.md

```diff
@@ -0,0 +1,143 @@
+# Aries New Secure KMS, SecretLock and Crypto capabilities
```

Comment:
Remove “Aries New Secure” from the title.

Reply:
done